helm/defectdojo/values.yaml

```diff
@@ -1,27 +1,25 @@
 ---
-# Global settings
-# create defectdojo specific secret
+## @section Global settings
+## @param createSecret create defectdojo specific secret
 createSecret: false
-# create rabbitmq secret in defectdojo chart, outside of rabbitmq chart
+## @param createRabbitMqSecret create rabbitmq secret in defectdojo chart, outside of rabbitmq chart
 createRabbitMqSecret: false
-# create redis secret in defectdojo chart, outside of redis chart
+## @param createRedisSecret create redis secret in defectdojo chart, outside of redis chart
 createRedisSecret: false
-# create mysql secret in defectdojo chart, outside of mysql chart
+## @param createMysqlSecret create mysql secret in defectdojo chart, outside of mysql chart
 createMysqlSecret: false
-# create postgresql secret in defectdojo chart, outside of postgresql chart
+## @param createPostgresqlSecret create postgresql secret in defectdojo chart, outside of postgresql chart
 createPostgresqlSecret: false
-# create postgresql-ha secret in defectdojo chart, outside of postgresql-ha chart
+## @param createPostgresqlHaSecret create postgresql-ha secret in defectdojo chart, outside of postgresql-ha chart
 createPostgresqlHaSecret: false
-# create postgresql-ha-pgpool secret in defectdojo chart, outside of postgresql-ha chart
+## @param createPostgresqlHaPgpoolSecret create postgresql-ha-pgpool secret in defectdojo chart, outside of postgresql-ha chart
```

Comment:
We need some documentation how to write this annotations.

Reply:
Sure https://github.com/bitnami-labs/readme-generator-for-helm

Reply:
I am not sure where the best place is, but we need to add this link with one or two lines of description in the values.yaml or in our docs.